src/components/v2/V2Create/tabs/ReviewDeployTab/FundingSummarySection.tsx

```diff
@@ -0,0 +1,220 @@
+import { t, Trans } from '@lingui/macro'
```

Comment:
This might be a nice to have, or a ticket we open for future, but we could probably reshare some of the parts of this component and the one in v2 reconfigure.

Reply:
Yeah indeed, was missing shared components for the splits but I've just made them

Reply:
🙌